tests/integration/issues/github_1229/test_sharding_empty_index.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import numpy as np
+
+from jina import Flow, Document
+
+PORT = 45888
+
+
+def test_sharding_empty_index(random_workspace_name):
```

Comment:
I showed you the `random_workspace_name`, but apparently, either we use plain pytest `tmpdir` fixture or we use `test_metas`. Since the second one is a bit overkill, I'd say, let' go for `tmpdir`. Main reason: The created directories are in the operating system temporary directory and not in the jina main directory.

Reply:
Done